reliability-v2/config/reliability-netobserv.yaml

```diff
@@ -78,10 +110,10 @@ reliability:
         - func apply -n 8 -p "<path_to_content>/http-service.yaml"
         - func apply -n 8 -p "<path_to_content>/http-route.yaml"
       tasks:
-        - func load_app -n 8 -p 50 
-        - func scale_deployment -n 8 -p 8 # scale deployment in 2 namespaces to 2 replicas
-        - func scale_deployment -n 8 -p 5 # scale deployment in 2 namespaces to 1 replicas
-      post_tasks: 
+        - func load_app -n 8 -p 50
+        - func scale_deployment -n 8 -p 2 # scale deployment in 2 namespaces to 2 replicas
+        - func scale_deployment -n 8 -p 1 # scale deployment in 2 namespaces to 1 replicas
```

Comment:
same comment about amount of load.

Reply:
@memodi Scaling too big here will cause a peak number of pods, as we have the 250 pod per node by default, if the peak is high, we have to reduce the base load, thus decrease the total workload. So this is a trade off, being more mild at scale can allow us to have more users/namespaces. With this new version of the load, I hope to firstly make it work (no pending pod), then we can adjust the balance between base load v.s. peak load according to your test request.